### PR TITLE
ENG-19 delete entity flow

### PR DESCRIPTION
**Pull request by gidesan**

### CODE REVIEW

File: react/table/src/components/ConferenceTableContainer.js [hunk 1]
```diff
@@ -139,9 +144,56 @@ class ConferenceTableContainer extends Component {
...
+    const { classes, onSelect, onAdd, onDelete, t, keycloak, paginationMode = '' } = this.props;
+    const deleteLabel = t('common.delete');
+
+    const Actions = ({ item }) =>
+      onDelete ? (
+        <ConfirmationDialogTrigger
+          onCloseDialog={action => this.handleConfirmationDialogAction(action, item)}
+          dialog={{
+            title: t('entities.conference.deleteDialog.title'),
+            description: t('entities.conference.deleteDialog.description'),
```
Comment (rimselis) [inline react/table/src/components/ConferenceTableContainer.js:184]: Maybe worth having these as common with placeholder variables?

File: react/table/src/components/common/ConfirmationDialogTrigger.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          aria-describedby="confirmation-dialog-description"
+        >
+          <DialogTitle id="confirmation-dialog-title">{title}</DialogTitle>
+          <DialogContent>
+            <DialogContentText id="confirmation-dialog-description">
+              {description}
+            </DialogContentText>
+          </DialogContent>
+          <DialogActions>
+            <Button onClick={() => this.handleClose(ConfirmationDialogTrigger.DISCARD)} autoFocus>
```
Comment (rimselis) [inline react/table/src/components/common/ConfirmationDialogTrigger.js]: They used to suggest avoiding creating functions inside render

**Review by rimselis — DISMISSED** [review]

File: react/table/src/components/ConferenceTableContainer.js (hunk 1 above)
Comment (gidesan) [reply react/table/src/components/ConferenceTableContainer.js:184]: I expect those values to be specific to conference

File: react/table/src/components/common/ConfirmationDialogTrigger.js (hunk 2 above)
Comment (gidesan) [reply react/table/src/components/common/ConfirmationDialogTrigger.js]: correct, thanks! updated

**Review by rimselis — APPROVED** [review]

**Pull request merged**